src/commands.js

```diff
@@ -148,128 +144,130 @@ commands.set("deposit", {
-          let backup_mnemonic = process.env.BACKUP_SEED_PHRASE;
-          let hash = await transferAccount(dest, dest_value, backup_mnemonic);
+          let hash = await transferAccount(dest, dest_value, mnemonic);
           const link = http_url + "#/explorer/query/" + hash;
           interaction.followUp({
             content: `Status: Complete
-      Amount:  ${dest_value} AVL
-      Block Hash: ${hash}
-      🌐 ${hyperlink("View in explorer", link)}`,
+    Amount:  ${dest_value} AVL
+    Block Hash: ${hash}
+    🌐 ${hyperlink("View in explorer", link)}`,
           });
         } catch (error) {
           console.error(error);
           interaction.followUp({
-            content: `There was a problem with the transfer. Kindly report to the Avail Team.`,
+            content: `testing it out!`,
             ephemeral: true,
           });
+          try {
+            // const amount = formatNumberToBalance(dest_value, decimals);
+            let backup_mnemonic = process.env.BACKUP_SEED_PHRASE;
+            let hash = await transferAccount(dest, dest_value, backup_mnemonic);
+            const link = http_url + "#/explorer/query/" + hash;
+            interaction.followUp({
+              content: `Status: Complete
+      Amount:  ${dest_value} AVL
+      Block Hash: ${hash}
+      🌐 ${hyperlink("View in explorer", link)}`,
+            });
+          } catch (error) {
+            console.error(error);
+            interaction.followUp({
+              content: `There was a problem with the transfer. Kindly report to the Avail Team.`,
+              ephemeral: true,
+            });
+          }
         }
-      }
-      // await api.tx.balances
-      //   .transfer(dest, amount)
-      //   .signAndSend(keyring, options, ({ status, txHash }) => {
-      //     console.log(`Transaction status: ${status.type}`);
-      //     if (status.isFinalized) {
-      //       const blockHash = status.asFinalized;
-      //       const link = http_url + "#/explorer/query/" + blockHash;
-      //       console.log(`transferred ${dest_value} AVL to ${dest}`);
-      //       console.log(`Transaction hash ${txHash.toHex()}`);
-      //       console.log(
-      //         `Transaction included at blockHash ${status.asFinalized}`
-      //       );
-      //       interaction.followUp({
-      //         content: `Status: Complete
-      //       Amount:  ${dest_value} AVL
-      //       Txn Hash: ${txHash}
-      //       Block Hash: ${blockHash}
-      //       🌐 ${hyperlink("View in explorer", link)}`,
-      //       });
-      //     }
-      //   });
-      const WeeklydepositInfo = await db
-        .collection("WeeklyRollupdepositInfo")
-        .findOne({ userId });
-      const DailydepositInfo = await db
-        .collection("DailyRollupdepositInfo")
-        .findOne({ userId });
-      if (WeeklydepositInfo) {
-        let { tokens } = WeeklydepositInfo;
-        await db
+        // await api.tx.balances
```

Comment:
Do we need this code now?
Can we delete it?

Reply:
yeh we don't 😅